server/methods/email.js

```diff
@@ -117,10 +117,25 @@ Meteor.methods({
     }
 
     check(jobId, String);
+    let emailJobId = jobId;
 
     Logger.debug(`emails/retryFailed - restarting email job "${jobId}"`);
 
-    Jobs.update({ _id: jobId }, {
+    // Get email job to retry
+    const job = Jobs.getJob(jobId);
+    // If this job was never completed, restart it and set it to "ready"
+    if (job._doc.status !== "completed") {
```

Comment:
We probably should add why we're adding this logic (to help future debugging), what do you think?
I mean the fact that we're doing this because jobs weren't getting resent successfully?

Reply:
@impactmass Do you not think the inline comments are enough, or just need more explanation overall?

Reply:
The current comment is good.... but I meant adding something around the reason why those steps are now needed. Something like what you have in the PR description.
`.....because of the way that the job-collection treats completed jobs. Only jobs that have failed or been canceled can be restarted. Completed jobs must be rerun.`